src/jit/hwintrinsicxarch.cpp

```diff
@@ -911,6 +911,36 @@ GenTree* Compiler::impSSEIntrinsic(NamedIntrinsic        intrinsic,
             retNode = gtNewSimdHWIntrinsicNode(TYP_VOID, intrinsic, TYP_VOID, 0);
             break;
 
+        case NI_SSE_SetVector128:
```

Comment:
It's probably not a good idea to handle this in the importer. Such intrinsics should be imported "as is" and expansion should be delayed until lowering/codegen so constant parameters can be better handled.

Reply:
Could you give me an example how lowering/codegen handles constant parameters better?

Reply:
See last commit and post in #14393 for handling SIMD's `Vector4` case.

Reply:
Thank you. Are you sure that the loading the whole constant vector from memory is always better than the current approach? I am afrid that `Avx.SetVector256` may have cache-line split issue.

Reply:
> Are you sure that the loading the whole constant vector from memory is always better than the current approach?

It's could be that it's not always better, it depends on the number of operands, how many are constants and what instructions are available. In some cases you end up with a lot of code that's going to be larger and slower than just loading from memory (assuming that the data is at least in L2).

Note that in the case of floating point constants will typically be loaded from memory anyway. Like it happens in the example from #17256.

> I am afrid that Avx.SetVector256 may have cache-line split issue.

Yeah, in that case we need 32 byte alignment support for data constants. Currently we only get 16 byte alignment.